src/js/ripple/amount.js

```diff
@@ -804,16 +836,19 @@ Amount.prototype.to_human = function(opts) {
     return 'NaN';
   }
 
+  /* eslint-disable consistent-this */
   // Apply demurrage/interest
   var ref = this;
```

Comment:
Could `ref` just be `this`? And if not, what is the `consistent-this` setting want? If it's just `self` instead of `ref` that shouldn't be too hard to change.